src/client/kotlin/tech/thatgravyboat/skycubed/api/overlays/Overlays.kt

```diff
@@ -114,6 +121,7 @@ object Overlays {
 
     init {
         register(PlayerRpgOverlay)
+        register(TrophyFishOverlay)
```

Comment:
I think we could add an @Overlay annotation to "auto load" these with the modules stuff